### PR TITLE
Test Symfony 5.3 compatibility

### PR DESCRIPTION
**Pull request by alexander-schranz**
| Q | A
| --- | ---
| Bug fix? | no 
| New feature? | no 
| BC breaks? | no 
| Deprecations? | no  
| Fixed tickets | fixes # 
| Related issues/PRs | # 
| License | MIT
| Documentation PR | sulu/sulu-docs# 

#### What's in this PR?

Test against minimum stability.

#### Why?

Test run against symfony 5.3

### CODE REVIEW

**Comment by alexander-schranz** [comment]
@sulu/core-team Seems like the only issue sulu did have with Symfony 5.3 was already fixed in https://github.com/symfony/symfony/pull/41009. Tests are running on `beta` version 🎉 . There is currently an issue with running on `dev` but this is related to `doctrine-bundle` and `doctrine/orm` 2.9 which they are already working on it  (https://github.com/doctrine/DoctrineBundle/pull/1327). So not related to the Symfony 5.3 version.

File: .github/workflows/test-application.yaml [hunk 1]
```diff
@@ -94,7 +94,7 @@ jobs:
                       dependency-versions: 'highest'
                       php-extensions: 'ctype, iconv, mysql, imagick'
                       tools: 'composer:v2'
-                      lint: true
+                      lint: false # disabled to ignore phpstan changes
```
Comment (alexander-schranz) [inline .github/workflows/test-application.yaml]: phpstan will need some work after Symfony 5.3 is released as they added better typehints to some classes we use, we can discuss doing a baseline or invest here some time add missing phpdocs.

Comment (StaffNowa) [reply .github/workflows/test-application.yaml]: Yes, Symfony 5.3 will be shorty released 🎉

Comment (alexander-schranz) [reply .github/workflows/test-application.yaml]: Mostly last day of may 🎉

Comment (StaffNowa) [reply .github/workflows/test-application.yaml]: Yes :) 🎉

**Comment by alexander-schranz** [comment]
Close as this is not longer needed and the above issues were fixed in Symfony before 5.3 was released. Also doctrine bundle is compatible after they release.

**Pull request closed without merging**